addons/html_builder/static/src/plugins/image/image_shape_option_plugin.js

```diff
@@ -406,6 +408,11 @@ export class SetImageShapeAction extends BuilderAction {
     static dependencies = ["imageShapeOption"];
     async load({ editingElement: img, value: shapeId }) {
         const params = { shape: shapeId };
+        if (this.shouldResetCrop(img, shapeId)) {
+            for (const field of cropperDataFieldsWithAspectRatio) {
+                params[field] = undefined;
+            }
```

Comment:
Maybe we can add a comment inside `if (this.shouldResetCrop(img, shapeId)) {` to explain that some shapes add a crop and that we want to remove it when removing the shape :+1: